src/index.js

```diff
@@ -0,0 +1,26 @@
+import './style.css';
+
+const taskInput = document.getElementById('input');
+const addButton = document.getElementById('button');
+const todoList = document.getElementById('todo-list');
+
+//  add work
+
+const removeTask = (event) => {
+  const li = event.target.parentNode;
+  todoList.removeChild(li);
+};
+
+const addTask = () => {
+  const taskValue = taskInput.value.trim();
+
+  const li = document.createElement('li');
+  li.innerHTML = `${taskValue} <button class="remove-button">X</button>`;
+  todoList.appendChild(li);
+
+  taskInput.value = '';
+  const removeButton = li.querySelector('.remove-button');
+  removeButton.addEventListener('click', removeTask);
+};
```

Comment:
- [ ] While you have made a brilliant feature functional such that it allows users to add to-do list items, I do not think this is a necessity for this milestone :clap:. A such you may need to check the requirement again to be sure of its expectations. According to the project requirement (shown below), you are expected to have an `array` of tasks as objects with some required `keys` pairs e.g. `description`, `completed`, and `index`. 

![image](https://user-images.githubusercontent.com/99715640/231842835-b7b3f7f7-17ee-472b-83f1-dd755928f8b9.png)
  - [ ] After this, it is expected that you create a function that traverses through or iterate the array in order to display each task according to your template and design, with much consideration for the `wireframe`/video in the project requirement.
  - [ ] And please, to make it really functional, do not forget to make the tasks get created in the UI when page loads according to their indexes/indices :wink:  
_Kindly implement these and submit them again for review_